openedx/features/content_type_gating/tests.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
i don't think we should add this file unless we add tests to it  
same for the migrations file